src/modules/eventHandle.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-return-assign */
+/* eslint-disable no-plusplus */
+import completed from './completed.js';
+import createList from './displayTasks.js';
+
+const todoList = document.querySelector('.todo-list');
+const form = document.querySelector('.form');
+const addTask = document.getElementById('add-task');
+const clear = document.querySelector('.clear');
+const claerRotate = document.querySelector('.fa-arrows-rotate');
+
+export const setData = (tasks) => {
+  localStorage.setItem('inputData', JSON.stringify(tasks));
+};
+
+const eventHandle = (tasks) => {
+  if (localStorage.getItem('inputData')) {
+    tasks = JSON.parse(localStorage.getItem('inputData'));
+    createList(tasks);
+  }
+  form.onsubmit = () => {
+    let index;
+    if (tasks.length > 0) {
+      index = tasks[tasks.length - 1].index + 1;
+    } else {
+      index = 1;
+    }
+    const task = addTask.value;
+    tasks.push({
+      description: task,
+      completed: false,
+      index,
+    });
+    createList(tasks);
+    form.reset();
+    setData(tasks);
+  };
+
+  todoList.onclick = (e) => {
+    if (e.target.className === 'fa-solid fa-trash-can') {
+      tasks = tasks.filter((item) => item.index !== parseInt(e.target.id, 10));
+      tasks.map((task, i) => task.index = i + 1);
+      createList(tasks);
+      setData(tasks);
+    }
+  };
+
+  todoList.onchange = (e) => {
+    completed(e, tasks);
```

Comment:
- Optional.
Since your previous reviewer didn't raise this as required, I will mark this as optional, however, I strongly recommend you implement this.
Any task marked with completed status should have a strike-through line on the content of the task. Consider implementing this.
Right now it is looking like this.
![image](https://user-images.githubusercontent.com/33268927/179177263-5acc3545-6912-4e0c-b88f-ea88469e8566.png)